src/media/headset.ts

```diff
@@ -0,0 +1,71 @@
+import { Observable } from 'rxjs';
+import GenesysCloudWebrtcSdk from '../client';
+import HeadsetService, { ConsumedHeadsetEvents, VendorImplementation} from 'softphone-vendor-headsets';
+
+export class SdkHeadset {
+    private sdk: GenesysCloudWebrtcSdk;
+    private headsetLibrary: HeadsetService;
+    headsetEvents$: Observable<ConsumedHeadsetEvents>;
+
+    constructor(sdk: GenesysCloudWebrtcSdk) {
+        this.sdk = sdk;
+        this.headsetLibrary = HeadsetService.getInstance({ logger: console });
```

Comment:
Do we want to pass in the sdk's logger or do we only ever want to log to the console?

Reply:
Probably the SDK's logger. I think I put this in originally while fleshing out the foundation